nion/swift/Panel.py

```diff
@@ -376,6 +395,108 @@ def context_menu_event(self, x: int, y: int, gx: int, gy: int) -> bool:
             return self.on_context_menu_clicked(x, y, gx, gy)
         return False
 
+
```

Comment:
Should only be a single blank line here. Please consider using `flake8 --ignore=E501 nion/swift/Panel.py nion/swift/DisplayPanel.py` and addressing any issues specific to this PR. You may need to install `conda install flake8` in your environment.